src/annotator/components/ContentInfoBanner.js

```diff
@@ -0,0 +1,44 @@
+import { LabeledButton, Link } from '@hypothesis/frontend-shared';
+
+/**
+ * @typedef {import('../../types/annotator').ContentInfoConfig} ContentInfoConfig
+ */
+
+/**
+ * A banner that informs the user about the provider of the document.
+ *
+ * @param {object} props
+ *   @param {ContentInfoConfig} props.info
+ *   @param {() => void} props.onClose
+ */
+export default function ContentInfoBanner({ info, onClose }) {
+  return (
+    <div className="flex items-center border-b gap-x-4 px-2 py-1 bg-white text-annotator-lg">
+      <Link href={info.logo.link} target="_blank">
+        <Link href={info.logo.link}>
+          <img src={info.logo.logo} alt={info.logo.title} />
```

Comment:
In this case, we're dependent on the `info.logo.logo` image being a reasonable size for this interface...which seems to be the case here, but is it dependable?